src/Solnet.Wallet/Utilities/BitWriter.cs

```diff
@@ -122,7 +122,7 @@ private static byte[] SwapEndianBytes(IReadOnlyList<byte> bytes)
                 }
                 output[i] = newByte;
             }
-            return output;
+            return output.ToArray();

```

Comment:
Same thing here, the array is returned, so a new copy is created in the heap. Revert to prevent more CPU usage in the copy and pointless stack allocation.